[2022-07-19 11:45:11,939] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:44:00+00:00 [queued]>
[2022-07-19 11:45:11,951] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:44:00+00:00 [queued]>
[2022-07-19 11:45:11,952] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:45:11,952] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 11:45:11,954] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:45:11,976] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 11:44:00+00:00
[2022-07-19 11:45:11,983] {standard_task_runner.py:52} INFO - Started process 12194 to run task
[2022-07-19 11:45:11,988] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T11:44:00+00:00', '--job-id', '634', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpqytueweq', '--error-file', '/tmp/tmpp_kuajg3']
[2022-07-19 11:45:11,989] {standard_task_runner.py:80} INFO - Job 634: Subtask respond
[2022-07-19 11:45:12,080] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:44:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 11:45:12,207] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T11:44:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T11:44:00+00:00
[2022-07-19 11:45:20,078] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 11:45:20,092] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 11:45:20,244] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T114400, start_date=20220719T114511, end_date=20220719T114520
[2022-07-19 11:45:20,338] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 11:45:20,487] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
